dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -161,7 +172,7 @@ class PkexecService(BaseService):
     """Class for parsing pkexec messages in the auth log"""
 
     PKEXEC_COMMAND_REGEX = re.compile(
-        r"(?P<user>.*?):\sExecuting\scommand\s"  # Starts with actual user -> user:
+        r"(?P<user>\S+?):\sExecuting\scommand\s"  # Starts with actual user -> user:
```

Comment:
Out of curiosity, why is this changed to `\S+?`? Is it because there can't be any spaces in usernames or are there cases where there is a `:` in the username for this message?